### ISSUE TITLE
Слетела лицензия

### ISSUE THREAD

**Issue by Akula1973** [dukei/any-balance-providers#1044]
Аккаунт PlayMarket: sergey.v.meshkov@gmail.com

**Comment by dukei** [comment]
Спасибо за поддержку!

Пожалуйста, сообщите ещё текущий серийный номер лицензии из меню Об
AnyBalance.



вс, 9 дек. 2018 г. в 22:58, Akula1973 <notifications@github.com>:

**Comment by Akula1973** [comment]
Раньше было:

 

Лицензирована на имя: sergey.v.meshkv@gmail.com <mailto:sergey.v.meshkv@gmail.com> 

Тип: Полная

Истекает: Пожизненная

S/N: 94AD1DO59BSZ

 

Теперь почему-то пишет:

 

Лицензирована на имя: 

Тип: Бесплатная

Истекает: Пожизненная

S/N: IT0B5LKXS1UB

 

Программу удалял и снова ставил из Маркета. Не помогло!

 

 

С уважением,

 

Сергей Мешков

**Comment by dukei** [comment]
Так это же другое устройство (хоть и та же модель телефона). У вас,
причем, на прежнем телефоне тоже работает программа.
К сожалению, лицензия на устройство распространяется только на одно устройство.
пн, 10 дек. 2018 г. в 17:21, Akula1973 <notifications@github.com>:

**Comment by Akula1973** [comment]
А разве у меня лицензия не привязана именно к аккаунту sergey <mailto:sergey.v.meshkov@gmail.com> .v.meshkov@gmail.com без ограничений?

 

С уважением,

 

Сергей Мешков

**Comment by dukei** [comment]
Нет. Вы можете посмотреть у себя в истории покупок на wallet.google.com.
если в названии лицензии есть слово устройство или device, значит, она
привязана к устройству

пн, 10 дек. 2018 г., 22:39 Akula1973 notifications@github.com:

**Comment by Akula1973** [comment]
Понятно, я оплачу ещё одну лицензию тогда. Напомните, не нужно вводить IMEI телефона? Я по-прежнему могу это сделать через Яндекс.Деньги?

 

С уважением,

 

Сергей Мешков

**Comment by dukei** [comment]
Да, не нужно. Можно оплатить лицензию Яндекс деньгами на сайте
https://anybalance.ru/shop

пн, 10 дек. 2018 г., 22:47 Akula1973 notifications@github.com:

**Comment by ivan100475** [comment]
Помогите с переносом лицензии на новый смартфон.

**Comment by dukei** [comment]
Добрый день!

Спасибо за поддержку!

У программы есть три вида лицензий - вечная с привязкой к устройству,
годовая с привязкой к гугл аккаунту и вечная с привязкой к гугл аккаунту.
Лицензии главным образом обеспечивают подписку на обновления
версий провайдеров и установку провайдеров из каталога.

Похоже, вы приобрели лицензию, привязанную к устройству, которая
по сути своей является временной (до смены аппарата). О привязке к
устройству в программе и на сайте (http://anybalance.ru/shop ) написано.

Вопрос о привязке задают часто, поэтому я даже поместил ответ на него
в теме, посвященной программе на 4pda.ru:

Q. Почему привязка к устройству? Очень не хочется программу каждый раз
при замене телефона покупать...

Программа бесплатна, её не нужно покупать. Она работает и получает
балансы независимо от того, полная у вас версия или нет. А вот платная
функциональность - это пользование каталогом и автоматическое
обновление версий провайдеров, то есть, те вещи, которые требуют от
меня _постоянных_ затрат и труда. Пользуясь программой, вы же хотите,
чтобы при изменении личного кабинета я тут же выпускал обновленную
версию провайдера, правильно? Хотя мне, может быть, тоже не хочется
это делать каждый раз...

Если смена устройства происходит слишком быстро (до 2х месяцев) - то я
переношу лицензию на новое устройство. Устройство с полной лицензией
имеет полные возможности программы и автоматические обновления
провайдеров в течение всей своей жизни. Перепрошивка, аппаратный сброс
лицензию сохраняет.

Если вы часто меняете устройства или у вас одновременно несколько
устройств, вам может быть удобно купить лицензию с привязкой к гугл
аккаунту. Она не зависит от устройств, важно только, чтобы они работали
под вашим гугл аккаунтом. Лицензия, привязанная к гугл аккаунту, может
быть годовой (с возможным автопродлением) или вечной (в этом случае
вам больше никогда не понадобится покупать лицензию на программу).

Надеюсь на ваше понимание

ср, 18 сент. 2019 г. в 22:50, ivan100475 <notifications@github.com>:

**Comment by Akula1973** [comment]
Прошу прощения, не понял, почему вдруг снова всплыла эта тема? Вопрос был закрыт ещё в прошлом году в декабре покупкой новой лицензии.

 

С уважением,

 

Сергей Мешков

**Comment by babar1905** [comment]
Добрый день !
Такая же история - перепрошил устройство (Android 10)/ Слетела лицензия. Покупал 02.11.2019
Счет № | 99132809 (выписан 02.11.2019 23:28:56)
SN сейчас: 3LQ3T4KBAZZBD

**Comment by vladislavik58** [comment]
У меня та же самая проблема что и остальных, слетела лицензия. Устройство не менял, прошивку не менял. Пишет бесплатная, пожизненная. Почините пожалуйста. Sn 1Y42VRGGVGGDG

**Comment by speedracerAlex** [comment]
Здравствуйте.
Слетела пожизненная лицензия для устройства. Помогите восстановить. 

![Screenshot_20200323-234421_AnyBalance](https://user-images.githubusercontent.com/62571452/77360900-41b62c00-6d5f-11ea-9dc8-e3de81172d59.jpg)

**Comment by dukei** [comment]
Лицензия восстановлена. Спасибо вам большое за поддержку!

пн, 23 мар. 2020 г. в 23:37, speedracerAlex <notifications@github.com>:

**Comment by lpvsnn** [comment]
Здравствуйте! 
Перестала работать пожизненная лицензия на устройство. Прошу восстановить. 
![Screenshot_2020-05-10-14-35-22-730_com dukei android apps anybalance](https://user-images.githubusercontent.com/63034729/81498142-dce17180-92cb-11ea-9fca-63ae9e58e927.jpg)

**Comment by agrik69** [comment]
Слетела лицензия. Что делать?

**Comment by dukei** [comment]
Добрый день!

Сообщите, пожалуйста, ваш гугл аккаунт, с которого вы покупали лицензию, а
также текущий статус и серийный номер лицензии из меню Об AnyBalance.

Спасибо Вам большое за поддержку!

чт, 28 мая 2020 г. в 18:25, agrik69 <notifications@github.com>:
